src/Common/AzurePSCmdlet.cs

```diff
@@ -334,11 +334,21 @@ protected virtual void TearDownHttpClientPipeline()
             AzureSession.Instance.ClientFactory.RemoveUserAgent(ModuleName);
             AzureSession.Instance.ClientFactory.RemoveHandler(typeof(CmdletInfoHandler));
         }
+
+        protected virtual void InitializePSStyle()
+        {
+            if (null == PSStyle.SupportsVirtualTerminal)
+            {
+                PSStyle.SupportsVirtualTerminal = Host.UI.SupportsVirtualTerminal;
+            }
```

Comment:
This logic is too specific to be put in AzurePSCmdlet. It should be in PSStyle.

Reply:
Host.UI.SupportsVirtualTerminal is a property of PSCmdlet, its value is unavailable beyond cmdlet.